data/templates/keyboard/rules.mk

```diff
@@ -1 +1 @@
-# This file intentionally left blank
+# This file is intentionally left blank
```

Comment:
If this is by intention then please feel free to close this PR.

Reply:
It's technically grammatically correct as is.
https://en.wikipedia.org/wiki/Intentionally_blank_page

Reply:
Ah nice, fine with me.